src/core/Object3D.js

```diff
@@ -409,10 +409,10 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		object.applyMatrix4( _m1 );
 
-		object.updateWorldMatrix( false, false );
-
 		this.add( object );
 
+		object.updateWorldMatrix( false, false );
```

Comment:
Children of `object` do not need to have their world matrices updated, right?

Reply:
Yeah, I don't think they need to be updated because `object.matrixWorld` values should not be changed then no effect to children's world matrices.

So, cutting off this `.updateWorldMatrix()` is another option. (I thought this calling is for safety just in case)

Reply:
Ah, let me change my opinion. If it's for safety just in case, we should update the children world matrices, too, with `.updateWorldMatrix( true, false )`.

So I think either cutting off this line or calling `.updateWorldMatrix( true, false )` would be good. Any thoughts?

Reply:
The signature is
```js
updateWorldMatrix( updateParents, updateChildren )
```
correct?

Reply:
Right, thanks for the correction. Updated the comment.